lib/DXIL/DxilModule.cpp

```diff
@@ -1021,8 +1030,9 @@ void DxilModule::RemoveUnusedResources() {
 
 namespace {
 template <typename TResource>
-static void RemoveResourcesWithUnusedSymbolsHelper(
+static bool RemoveResourcesWithUnusedSymbolsHelper(
     std::vector<std::unique_ptr<TResource>> &vec) {
+  bool modif = false;
```

Comment:
The name `modif` has no precedent in this code base, we should use `Changed` instead. Elsewhere we might have used `bChanged`, but we have decided to try to follow our latest coding standards for new code wherever we can, even if it's in an existing function like this, as there isn't truly a single simple coding standard being followed by the existing code.
```suggestion
  bool Changed = false;
```